stdlib/nodejs/file.rb

```diff
@@ -26,6 +26,79 @@
     warnings[string] = true;
     #{warn(`string`)};
   }
+  function is_utf8(bytes) {
```

Comment:
Code climate is unhappy: "File file.rb has 308 lines of code (exceeds 250 allowed). Consider refactoring."
Should I move this function somewhere else ? Do you have a better idea on how to handle both binary and utf8 encoded data without using this method ?

Reply:
CodeClimate "errors" should be taken as just suggestions, I regularly go and silence that stuff, no worry 👍